cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -123,9 +123,14 @@ describe('Create scenario', () => {
       }
     );
 
+    // Check parameters accordion state after creating scenario
+    ScenarioParameters.getParametersTabs().should('be.visible');
+    ScenarioManager.switchToScenarioManager();
+    Scenarios.switchToScenarioView();
+    ScenarioParameters.getParametersTabs().should('be.visible');
+
     // Edit master paramameters values
```

Comment:
Not your code, but could you fix the typo here please ? :grin:

Reply:
In fact    ` ScenarioParameters.getParametersTabs().should('be.visible');` allow to check if the accordion is correctly expand.